spec-main/api-app-spec.ts

```diff
@@ -891,53 +892,37 @@ describe('app module', () => {
       expect(app.isDefaultProtocolClient(protocol)).to.equal(false);
     });
 
-    it('creates a registry entry for the protocol class', (done) => {
+    it('creates a registry entry for the protocol class', async () => {
       app.setAsDefaultProtocolClient(protocol);
 
-      classesKey.keys((error: Error, keys: any[]) => {
-        if (error) throw error;
-
-        const exists = !!keys.find(key => key.key.includes(protocol));
-        expect(exists).to.equal(true);
-
-        done();
-      });
+      const keys = await promisify(classesKey.keys).call(classesKey) as any[];
```

Comment:
@miniak why does this need to be async/await? It doesn't appear to be an async operation?

Reply:
Oops... I missed the fact that it was callback based before... nevermind.